monkey/monkey_island/cc/services/authentication_service/otp_generator.py

```diff
@@ -0,0 +1,16 @@
+from .authentication_facade import AuthenticationFacade
+
+
+class OTPGenerator:
+    """
+    Generates OTPs
+    """
+
+    def __init__(self, authentication_facade: AuthenticationFacade):
+        self._authentication_facade = authentication_facade
+
+    def generate_otp(self) -> str:
```

Comment:
```suggestion
    def generate_otp(self) -> OTP:
```
Don't forget this in the `IOTPGenerator`